Posts/2021/02/XML-NewEmployees.md

```diff
@@ -0,0 +1,180 @@
+```yaml
+---
+post_title: 'Leveraging XML with New Employee PowerShell Scripts'
+username: s-r-turner
+categories: ActiveDirectory, XML, PowerShell
+tags: XML, ActiveDirectory
+featured_image:
+summary: using xml referencing to bolster up your new employee scripts with information.
```

Comment:
summary: Using XML referencing to bolster up your new employee scripts with information.